### DISCUSSION TITLE
Annotation for grounded caption task.

### DISCUSSION THREAD

**Discussion by Kingofolk** [wkentaro/labelme#1398] [Ideas / Feature requests]
Hi, is there any ways to accompolish the annotation of grounded caption? If not, could this function be incoportated? Thx a lot!
best regards

**Comment by wkentaro** [comment]
Like AI-mask, AI-polygon, but with text not a point?

SAM model supports that, which is already used in Labelme, so this should be close to possible.

**Reply by Kingofolk** [reply]
Yes, u r right. Grounded caption annotation aims to write a paragragh to describe the image, then grounded each valuable noun phrase on the original picture. Could you give me a path to know how to achieve this via labelme? As I do not find tutorial about this on the intro page.  Thanks a lot!
best regards
Kingofolk
